### ISSUE TITLE
Missing `download_shapefile` function? 

### ISSUE THREAD

**Issue by hannahker** [OCHA-DAP/hdx-signals#89]
On running `src-static/update_cities_sf.R`, I get the following error: 

```
Error in box::use(./utils/download_shapefile[download_shapefile]) : 
  unable to load module “./utils/download_shapefile”; not found in “./src-static”
```

Can't seem to find where this function is defined.

**Comment by zackarno** [comment]
it should be

`box::use(../src/utils/download_shapefile[download_shapefile])`

**Comment by zackarno** [comment]
I feel like it would be much easier if all paths in box could be in reference to the `.rproj` root directory. Then it would be much easier to figure out where files are.

Perhaps this is antithetical to box though?

**Comment by caldwellst** [comment]
Not normal practice, all are relative references. See [here](https://github.com/klmr/box/issues/322). I pushed a fix in #80.

**Comment by hannahker** [comment]
Closed with #80
